hugegraph-test/src/main/java/com/baidu/hugegraph/unit/core/UnitTestSuite.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2017 HugeGraph Authors
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with this
+ * work for additional information regarding copyright ownership. The ASF
+ * licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.baidu.hugegraph.unit.core;
+
+import com.baidu.hugegraph.util.StringUtil;
+import org.junit.Assert;
+import org.junit.Test;
+import java.util.ArrayList;
+
+public class UnitTestSuite {
```

Comment:
I mean that this class should be registered to [UnitTestSuite](https://github.com/hugegraph/hugegraph/blob/master/hugegraph-test/src/main/java/com/baidu/hugegraph/unit/UnitTestSuite.java#L48) in order to be run by mvn test. Please feel free to change the class name back to StringUtilTest.

Reply:
Apologies I have changed the class name back and added  `StringUtilTest.class,` to hugegraph-test/src/main/java/com/baidu/hugegraph/unit/UnitTestSuite.java